Worker.h

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2021 Chiu Yen-Chen, Swen Sun-Yen, Wen Yong-Wei, Yuan Wei-Chen.
+All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. See the AUTHORS file for names of contributors.
+ */
+#ifndef SHORTLINK_SRC_WORKER_H_
+#define SHORTLINK_SRC_WORKER_H_
+#include <string>
+
+namespace shortlink
+{
+    //ThreadPool will create this class 
+    //Example:
+    // Worker worker = Worker();
+    // worker.Run("RawRequestData");
+    class Worker
+    {
+    private:
+        //worker status 0:buzy 1:free 1:Done
+        int worker_Status;
```

Comment:
Use lower case for naming: `worker_status`
https://google.github.io/styleguide/cppguide.html#Variable_Names